app/org/zalando/markscheider/MetricsController.scala

```diff
@@ -2,26 +2,70 @@ package org.zalando.markscheider
 
 import java.io.StringWriter
 
-import play.api.mvc.{ Action, AnyContent, InjectedController, Result }
+import com.codahale.metrics.MetricRegistry
 import com.fasterxml.jackson.databind.{ ObjectMapper, ObjectWriter }
 import com.google.inject.Inject
+import play.api.Configuration
 import play.api.http.{ HeaderNames, MimeTypes }
+import play.api.mvc._
 
-class MetricsController @Inject() (plugin: MetricsPlugin) extends InjectedController {
+import scala.collection.JavaConverters._
 
-  def serialize(mapper: ObjectMapper): Result = {
+class MetricsController @Inject() (
+    plugin:        MetricsPlugin,
+    configuration: Configuration,
+    cc:            ControllerComponents
+)
+  extends AbstractController(cc) with PrometheusSupport {
+  private def defaultFormat: String = configuration.getAndValidate[String]("org.zalando.markscheider.defaultFormat", Set("zmon", "prometheus"))
+
+  def serializePrometheus(registry: MetricRegistry): Result = {
```

Comment:
This one could be private, right?
```suggestion
  private def serializePrometheus(registry: MetricRegistry): Result = {
```

Reply:
absolutely.